third_party/xla_client/pjrt_computation_client_test.cc

```diff
@@ -1,4 +1,6 @@
-#include "tensorflow/compiler/xla/xla_client/pjrt_computation_client.h"
+#include "third_party/xla_client/pjrt_computation_client.h"
+
+#include <gtest/gtest.h>
```

Comment:
We have `gtest` pin dependency in PyTorch/XLA. Does it make sense to transition this dependency to bazel also?

Reply:
Yes, but since that touches both `torch_xla` and the `libxla_cc_.so` I'd migrate that with the last part when we remove cmake. wdyt?

Reply:
Once we migrate the tests to bazel, we should just depend on the pinned gtest that comes in as an external repository from tensorflow and not have cmake do it.